### PR TITLE
Fix linting errors; add linting to Travis

### PR DESCRIPTION
**Pull request by tbroadley**
This PR resolves all remaining linting errors, and adds Javascript linting to Travis. The remaining errors were caused by variable declarations that were not at the top of the scope in which the variable was declared.

### CODE REVIEW

File: dwitter/static/js/ajax-handling.js [hunk 1]
```diff
@@ -25,54 +24,59 @@ var processLike = function(e) {
...
+function loadComments() {
   var step = 1000;
   var currentOffset = $(this).data('offset');
 
   var $loadCommentsButton = $(this);
 
   var dweetId = $loadCommentsButton.data('dweet_id');
   // If there is a sticky comment on the top of the comments
-  var stickyTop = $loadCommentsButton.data('stickyTop');
+  var stickyTop = $loadCommentsButton.data('sticky_top');
```
Comment (tbroadley) [inline dwitter/static/js/ajax-handling.js]: I believe this fixes a bug where the wrong data attribute was being read off of the button. 😨

Comment (lionleaf) [inline dwitter/static/js/ajax-handling.js]: Nice catch!

**Comment by tbroadley** [comment]
Looks like there's a pretty serious merge conflict to sort out. Unfortunately, I don't have my laptop on me. I'll fix the conflict next week!

**Comment by lionleaf** [comment]
As the merge conflict was introduced by my #190, I'll fix it :)

**Pull request merged**